backend/src/youi-adapter.ts

```diff
@@ -45,7 +45,7 @@ export class YouiAdapter extends TerminalAdapter {
   ): Promise<T2> {
     if (this.yeomanui && questions) {
       const result: any = await (this.yeomanui.showPrompt(questions) as Promise<T2>);
-      if (!_.isEmpty(cb)) {
+      if (!_.isNil(cb)) {
```

Comment:
I think we should also update/add test for param cb=null

Reply:
When use isFunction cb === null is already checked